addons/project/models/project_task.py

```diff
@@ -205,6 +205,10 @@ def _read_group_personal_stage_type_ids(self, stages, domain, order):
     partner_id = fields.Many2one('res.partner',
         string='Customer', recursive=True, tracking=True, compute='_compute_partner_id', store=True, readonly=False,
         domain="['|', ('company_id', '=?', company_id), ('company_id', '=', False)]", )
+    # Used to submit tasks from a contact form
+    partner_name = fields.Char(string='Customer Name', related="partner_id.name", store=True, readonly=False)
+    partner_phone = fields.Char(string='Customer Phone', related="partner_id.phone", store=True, readonly=False)
+    partner_company_name = fields.Char(string='Company Name', related="partner_id.company_name", store=True, readonly=False)
```

Comment:
Since those fields are only useful when `website_project` module, why not adding them in `website_project` module?